### DISCUSSION TITLE
Projects (Beta) [BUG]: Date formatting on cards

### DISCUSSION THREAD

**Discussion by MichielDeMey** [community/community#5525] [Projects and Issues]
Looks like a custom "Date" field shows as a full date in the table view, but as an ISO timestamp on the cards in the kanban view.

I assume both should just display the date without a time.

<img width="343" alt="Screenshot 2021-09-08 at 15 18 01" src="https://user-images.githubusercontent.com/793406/132517002-65eaab88-0d5e-445f-a2ff-ef9eacbeb94a.png">
<img width="424" alt="Screenshot 2021-09-08 at 15 20 22" src="https://user-images.githubusercontent.com/793406/132517092-ed972f18-48f9-4298-8d28-f6df062acc48.png">

**Comment by jenakal** [comment]
Also wanted to know this[.](https://einternetpackages.com/)
